src/overlays/actors/ovl_Structure/ac_structure.h

```diff
@@ -4,15 +4,65 @@
 #include "ultra64.h"
 #include "m_actor.h"
 #include "unk.h"
+#include "c_keyframe.h"
 
 struct Game_Play;
 struct Structure;
+struct Game;
 
 typedef void (*StructureActionFunc)(struct Structure*, struct Game_Play*);
 
 typedef struct Structure {
     /* 0x000 */ Actor actor;
-    /* 0x174 */ UNK_TYPE1 unk_174[0x8];
-} Structure; // size = 0x17C
+    /* 0x174 */ s32 unk174;
+    /* 0x178 */ SkeletonInfoR skeletonInfo;
+    /* 0x1E8 */ s32 unk1E8;
+    /* 0x1EC */ Vec3s jointTable[15];
+    /* 0x246 */ Vec3s morphTable[15];
+    /* 0x2A0 */ StructureActionFunc unk2A0;
+    /* 0x2A4 */ UNK_TYPE1 unk2A4[0x10];
+    /* 0x2B4 */ UNK_TYPE unk2B4;
+    /* 0x2B8 */ UNK_TYPE1 unk2B8[0x4];
+    /* 0x2BC */ UNK_TYPE unk2BC;
+    /* 0x2C0 */ UNK_TYPE1 unk2C0[0x8];
+    /* 0x2C8 */ f32 unk2C8;
+    /* 0x2CC */ f32 unk2CC;
+    /* 0x2D0 */ f32 unk2D0;
+    /* 0x2D4 */ UNK_TYPE1 unk2D4[0x4];
+} Structure; // size = 0x2D8
```

Comment:
I think I didn't explain myself properly.
Instead of changing this struct and making it bigger I was trying to say to change the `Toudai` struct, so it used `Structure` as a base, like this
```c

typedef struct Toudai {
    /* 0x000 */ Structure structure;
    /* 0x17C */ the other members...
} Toudai; // size = 0x2D8
```

We know the correct size of each actor struct from their profiles, so we shouldn't be changing them.

But now that I write this, I realize this may not be possible, because `Toudai` has a `SkeletonInfoR` at offset `0x178`, which should be part of `Structure`, but that does not fit the size of the `Structure` actor... yikes. I'm not sure what's going on with this actor, but it may be better to not merge the two for now